src/main/java/com/booleanuk/core/Basket.java

```diff
@@ -1,5 +1,66 @@
 package com.booleanuk.core;
 
+import java.util.ArrayList;
+
 public class Basket {
+    ArrayList<String> basket;
+    int basketCapacity;
+
+    public Basket(int basketCapacity) {
+        this.basket = new ArrayList<>();
+        this.basketCapacity = basketCapacity;
+    }
+
+/*
+        1.
+    As a member of the public,
+    So I can order a bagel before work,
+    I'd like to add a specific type of bagel to my basket.
+ */
+    public boolean addBagel(String bagel) {
+        if (this.basket.size() >= this.basketCapacity)
+            return false;
+
+        if (!this.basket.contains(bagel)) {
```

Comment:
Shouldn't this method return false inside the second if statement when the bagel already exists in the basket?